avaframe/data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini

```diff
@@ -15,6 +15,9 @@ rhoEnt = 200
 massPerPart = 150
 # minimum flow depth [m]
 hmin = 0.05
+#+++++++++++++SPH parameters
+# SPH gradient option
+sphOption = 4
```

Comment:
please add a comment which one this is - or TODO needs to updated if we have a 'final'one